src/client/app/components/RouteComponent.jsx

```diff
@@ -72,6 +73,7 @@ export default class RouteComponent extends React.Component {
 					<Route path="/login" component={LoginContainer} />
 					<Route path="/admin" component={AdminComponent} onEnter={this.requireAuth} />
 					<Route path="/groups" component={GroupContainer} onEnter={this.requireAuth} />
+					<Route path="/editGroup" component={EditGroupComponent} />
```

Comment:
Please remove this. This was removed in groupUI, but somehow not in your branch.